test/check-machines-create

```diff
@@ -1545,6 +1545,14 @@ vnc_password= "{vnc_passwd}"
             if not (b.is_present("#vm-details") and b.is_visible("#vm-details")):
                 self.test_obj.goToVmPage(name, dialog.connection or "system")
 
+            # Check connection information in the details page
+            if dialog.connection == "system":
+                wait(lambda: "System" == b.text(f"#vm-{dialog.name}-connection"))
+            elif dialog.connection == "session":
+                wait(lambda: "User session" == b.text(f"#vm-{dialog.name}-connection"))
+            else:
+                raise Exception("unsupported connection")
+
```

Comment:
We have a function `wait_in_text()` which does the same as `wait(lambda: [string] == b.text(...)`. And with conditional expression resolution this can be a one liner:
```suggestion
            # Check connection information in the details page
            `wait_in_text(f"#vm-{dialog.name}-connection", "System" if dialog.connection == "system" else "User session")`
```
I don't think it's necessary to test for other types of connection, as libvirt only supports those 2, either you run the VM instance as privileged user, or as unprivileged user.

Reply:
OK